addons/modules/XEH_postInit.sqf

```diff
@@ -79,17 +79,25 @@ if (isServer) then {
     // For small spread values, use doArtilleryFire directly to avoid delay
     // between firing caused by using doArtilleryFire one round at a time
     if (_spread <= 10) exitWith {
-        _unit doArtilleryFire [_position, _ammo, _rounds];
+        if (_unit isKindOf CLASS_VLS_BASE) then {
```

Comment:
VLS is handled twice in this. At the start just check if the unit is VLS and `exitWith` call to `fireVLS`, otherwise same as old.